variables.tf

```diff
@@ -280,6 +280,85 @@ variable "geo_match_statement_rules" {
   DOC
 }
 
+variable "label_match_statement_rules" {
```

Comment:
Nit: move `label_match_statement_rules` under `ip_set_reference_statement_rules` to maintain rules block alphabetical order.